src/js/components/Anchor.js

```diff
@@ -33,7 +33,8 @@ const Anchor = props => {
     {
       [`${CLASS_ROOT}--disabled`]: props.disabled,
       [`${CLASS_ROOT}--icon`]: icon,
-      [`${CLASS_ROOT}--primary`]: props.primary
+      [`${CLASS_ROOT}--primary`]: props.primary,
+      [`${CLASS_ROOT}--icon-label`]: icon && props.label && !props.primary
```

Comment:
Why do we need the !props.primary here? It looks like we only ever style it the same as primary.